odata-protocol/11 Data Service Requests.md

```diff
@@ -555,9 +555,9 @@ properties that can be used in a [`$select`](#SystemQueryOptionselect)
 or within a [`$filter`](#SystemQueryOptionfilter) or
 [`$orderby`](#SystemQueryOptionorderby) expression.
 
-Computed properties SHOULD be included as dynamic properties in the
-result and MUST be included if `$select` is specified with the computed
-property name, or star (`*`).
+Computed properties SHOULD be included as dynamic properties in the result.
+Computed properties whose value is a stream, an entity, or a collection of entities MUST be included if `$expand` is specified with the computed property name or star (`*`).
```

Comment:
`$expand=*` does not expand stream properties, they must be explicitly named.

Reply:
Discussion 2025-3-12: Expanding streams was added after expand=*. The server is always allowed to expand more than requested, but requiring expand=* to include streams is arguably not backward compatible.

Most compatible solution would be to continue the current definition of expand=* not including streams; streams must be explicitly enumerated.